lib/membrane_live_web/router.ex

```diff
@@ -15,13 +15,21 @@ defmodule MembraneLiveWeb.Router do
   end
 
   scope "/", MembraneLiveWeb do
+    # TODO add Plug for checking jwt and g_csrf
     pipe_through(:browser)
 
     get("/", PageController, :index)
     resources("/webinars", WebinarController, except: [:edit, :new], param: "uuid")
     get("/event/*page", PageController, :index)
   end
 
+  scope "/auth", MembraneLiveWeb do
+    pipe_through(:browser)
+
+    get("/", LoginController, :index)
+    post("/login", LoginController, :create)
```

Comment:
Do we need both routes?

Reply:
we need post and get, but it can be simplified to `/` I think 🤔